asset_dashboard/routers.py

```diff
@@ -0,0 +1,49 @@
+class GISRouter:
```

Comment:
This tells Django which DB to route ORM queries to. See: https://docs.djangoproject.com/en/3.2/topics/db/multi-db/#automatic-database-routing